aptos-node/src/bootstrap.rs

```diff
@@ -0,0 +1,56 @@
+use std::sync::Arc;
+
+use anyhow::anyhow;
+use anyhow::Result;
+use aptos_config::config::NodeConfig;
+use aptos_db::AptosDB;
+use aptos_storage_interface::DbReaderWriter;
+use tokio::runtime::Runtime;
+
+#[cfg(not(feature = "consensus-only"))]
```

Comment:
Can we not inline the feature gating (inside the method)? It seems like there's only a small difference between these two?